client/src/utils/dateFunctions.js

```diff
@@ -5,4 +5,22 @@ const formatDate = (date) => {
   return displayDate.toLocaleDateString(undefined, dateFormat);
 };
 
-export { formatDate };
+const formatDateAndTime = (dateTime) => {
+  const currentDatetime = new Date(dateTime);
+  const formattedDate =
+    currentDatetime.getFullYear() +
+    '-' +
+    (currentDatetime.getMonth() + 1) +
+    '-' +
+    currentDatetime.getDate() +
+    ' ' +
+    currentDatetime.getHours() +
+    ':' +
+    currentDatetime.getMinutes() +
+    ':' +
+    currentDatetime.getSeconds();
+
+  return formattedDate;
+};
```

Comment:
Wouldn't be using this better? 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleString